include/minit-assets.php

```diff
@@ -230,21 +230,17 @@ protected function get_asset_relative_path( $handle ) {
 			return false;
 		}
 
-		// Remove protocol reference from the local base URL
-		$base_url = preg_replace( '/^(https?:)/i', '', $this->handler->base_url );
+   		 $arr = explode(plugins_url(), $item_url);
```

Comment:
@strarsis What about plugins and WP core files? `plugins_url()` works only with assets in plugins.